### PR TITLE
Classifier probabilities & small fixes

### PR DESCRIPTION
**Pull request by jaseweston**

### CODE REVIEW

File: parlai/core/torch_agent.py [hunk 1]
```diff
@@ -720,7 +720,7 @@ def __init__(self, opt: Opt, shared=None):
         self.rank_candidates = opt['rank_candidates']
         self.add_person_tokens = opt.get('person_tokens', False)
         # set interactive mode or not according to options.
-        self.set_interactive_mode(opt['interactive_mode'], shared)
+        self.set_interactive_mode(opt.get('interactive_mode', False), shared)
```
Comment (emilydinan) [inline parlai/core/torch_agent.py]: when is interactive mode not set?

File: parlai/scripts/eval_wordstat.py [hunk 2]
```diff
@@ -112,6 +112,7 @@ def eval_wordstat(opt, print_parser=None):
     :param print_parser: if provided, prints the options that are set within the
         model after loading the model
     """
+    print(opt)
```
Comment (emilydinan) [inline parlai/scripts/eval_wordstat.py]: did you mean to add this here?

File: parlai/core/torch_agent.py (hunk 1 above)
Comment (jaseweston) [reply parlai/core/torch_agent.py]: hmm, i ran into this, but can't remember where now, srry

File: parlai/scripts/eval_wordstat.py (hunk 2 above)
Comment (jaseweston) [reply parlai/scripts/eval_wordstat.py]: hmm.. sort of did, sort of didn't.. haha..i'll remove

**Pull request merged**